src/server/binary/binaryServerStarter.ts

```diff
@@ -206,13 +206,32 @@ function getBinaryEnvironment(): any {
   return process.env;
 }
 
+/**
+ * Returns the platform of the current operating system.
+ * If the OS is Alpine Linux, it returns 'alpine'.
+ * Otherwise, it returns the platform string provided by the 'os' module.
+ *
+ * @returns The platform of the current operating system.
+ */
+function getPlatform (): ReturnType<typeof os.platform> | 'alpine' {
```

Comment:
Could this be `: NodeJS.Platform | 'alpine'` or do you feel it's more correct to reference the method itself since that's what is really being called ?